packages/network/src/v2/schemas/tableSchemas.ts

```diff
@@ -45,7 +46,7 @@ export function registerSchema(world: Contract, table: TableId, rawSchema?: stri
   // TODO: populate from ECS cache before fetching from RPC
 
   console.log("fetching schema for table", { table: table.toString(), world: world.address });
-  const schema = world.getSchema(table).then((rawSchema: string) => {
+  const schema = (world as IStore).getSchema(table.toHexString()).then((rawSchema: string) => {
```

Comment:
good catch!

Reply:
should we make the `world` argument an `IStore` here too?

Reply:
I tried that first but it requires to change it all the way upstream too, so I waited for now bc it's mostly unrelated to this PR. Just thought having type hints where we call the World methods is useful